tmt/steps/provision/artemis.py

```diff
@@ -548,7 +548,26 @@ def _create(self) -> None:
         # TODO: spot instance
 
         if self.post_install_script:
-            data['post_install_script'] = self.post_install_script
+            if tmt.utils.is_url(self.post_install_script):
+                try:
+                    with retry_session() as session:
+                        response = session.get(self.post_install_script)
+
+                    if not response.ok:
+                        raise ArtemisProvisionError(
+                            "Failed to get the post_install_script file")
```

Comment:
How about 'download' instead of get?

Reply:
And let's be more civil and consistent too: there is no such thing as "post_install_script" in what the user can see. They can see a CLI option, `--post-install-script`, or an fmf key, `post-install-script`; a name with underscores is a leak of an internal variable or key which muddies the waters. How about "Failed to download the post-install script."?

Reply:
The same happens on line 562, BTW.